Kinvey/Kinvey/FileStore.swift

```diff
@@ -89,6 +93,10 @@ open class FileStore<FileType: File> {
         return FileStore<FileType>(client: client)
     }
     
+    /**
+     Constructor that takes a specific `Client` instance or use the
```

Comment:
More useful to describe what it constructs - "Creates a new file cache on disk, at the path specified by `client.fileURL()`.